h/api/test/search_test.py

```diff
@@ -0,0 +1,324 @@
+"""Tests for h/api/search.py."""
```

Comment:
Is this docstring really adding anything? We have a strong convention on test module naming. This just seems like noise to me.